TLM/TLM/U/TextureUtil.cs

```diff
@@ -131,16 +134,15 @@ public static class TextureUtil {
             return ret;
         }
 
-        public static UITextureAtlas FindAtlas(string name) {
-            UITextureAtlas[] atlases =
-                Resources.FindObjectsOfTypeAll(typeof(UITextureAtlas)) as UITextureAtlas[];
+        public static UITextureAtlas FindAtlasOrNull(string name) {
+            UITextureAtlas[] atlases = Resources.FindObjectsOfTypeAll<UITextureAtlas>();
```

Comment:
I don't like it (very slow), especially when hidden behind property getter, but outside of scope of this PR. Don't touch 😄

Reply:
its not performance critical part of the code. all this increases loading time by fraction of a second.